peachjam/adapters/adapters.py

```diff
@@ -141,7 +141,10 @@ def update_document(self, url):
 
         if frbr_uri.actor:
             frbr_uri_data["frbr_uri_actor"] = frbr_uri.actor
-            field_data["author"] = Author.objects.get(code__iexact=frbr_uri.actor)
+            author, _ = Author.objects.get_or_create(
```

Comment:
presumable it know to use `code=frbr_uri.actor` when creating the missing entry?

Reply:
@longhotsummer  Thanks for noting that. I have changed. Apparently having  `__iexact` results in the code not being added!